src/client/Microsoft.Identity.Client/Internal/Requests/Silent/SilentClientAuthStrategy.cs

```diff
@@ -258,7 +257,7 @@ private async Task<MsalRefreshTokenCacheItem> FindRefreshTokenOrFailAsync()
 
         private async Task<IAccount> GetSingleAccountForLoginHintAsync(string loginHint)
         {
-            var accounts = await CacheManager.GetAccountsAsync(ServiceBundle.Config.AuthorityInfo.CanonicalAuthority)
+            var accounts = await CacheManager.GetAccountsAsync()
```

Comment:
Small refactoring - there's no need to pass the authority here, the cache manager has a reference to the service bundle.